internal/charmstore/migrations_integration_test.go

```diff
@@ -384,6 +395,14 @@ var migrationFromDumpBaseEntityTests = []struct {
 				Read:  []string{"bobgroup"},
 				Write: []string{"bob", "someoneelse"},
 			},
+			params.BetaChannel: {
+				Read:  []string{"bobgroup"},
+				Write: []string{"bob", "someoneelse"},
+			},
```

Comment:
ISTM that it would be good to have an entity with different unpublished and edge ACLs so that we're verifying that the new channels are populated from the correct source.